index.html

```diff
@@ -27,40 +26,46 @@ <h4>Somerset based Web Developer</h4>
 			</section>
 			<div class="headImage">
 				<div class="buttonHolder">
-					<div>
-						<a href="#bigBox">
-							<h2>View Portfolio</h2>
-						</a>
-					</div>
+					<a href="#bigBox">
+						<h2>View Portfolio</h2>
+					</a>
 				</div>
 
-				<h3 id="about">After spending all of my career in Customer Service,
-					I am studying with Mayden Academy to become a Full-stack Web Developer.
-					By the end of the course I will be proficient in the use of HTML, CSS, PHP,
-					Javascript and MySql as well as being a Certified Scrum Master.
+				<h3 id="about">My name is Kyam Harris, a father of two from
+					Somerset currently training to become a Full Stack Web
+					Developer with Mayden Academy.
+					I’ve spent my whole career in Customer Service, but recently
+					discovered that I enjoyed using computers to solve problems
+					at work and, after tinkering with Python, that I enjoy programming!
+					<br><br>
```

Comment:
`<br>` tags, better practice to use additional <p> tag

Reply:
Looks like something is missing?

Reply:
resolved